js/libs.js

```diff
@@ -347,7 +357,9 @@ Fliplet.Registry.set('notification-inbox:1.0:core', function(element, data) {
           category: 'notification_inbox',
           action: 'notification_read_all'
         });
-        markAllAsRead();
+        markAllAsRead().then(function() {
+          $('.notification-read').hide();
```

Comment:
We should only be hiding notifications as read if it's toggled to only show unread notifications.